README.md

```diff
@@ -1,24 +1,151 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 Hello-World <a name="about-project"></a>
 
-* System dependencies
+**Hello-World**
+Now that you're familiar with Rails and React, it's time to put them together in a new kind of 'Hello World!' app. This exercise is going to have you create a React front-end with a Rails back-end and connect them to display a random message.
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a>
 
-* Database creation
+Rails
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+<details>
+  <summary>Client</summary>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Server</summary>
+</details>
 
-* Deployment instructions
+<details>
+<summary>Database</summary>
+</details>
 
-* ...
+### Key Features <a name="key-features"></a>
+
+- **Create a new comments**
+- **Get it display in json format**
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Command line or terminal with Rails installed.
+- IDE such as VsCode.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  https://github.com/Alabi12/hello-rails-back-end.git
+```
+```
+  React front End
+
+  https://github.com/Alabi12/hello-react-front-end.git
+```
+
+## Link to Hello_React_Front_End (http://localhost:3001)
+
```

Comment:
- [ ] [OPTIONAL] Kindly remove this url as you have already indicated the correct URL to the Front-end design.